pkg/base-dev/lib/ethereum.hoon

```diff
@@ -942,6 +942,15 @@
   %-  render-hex-bytes
   (as-octs:mimes:html n)
 ::
+++  num-to-hex-minimal
+  |=  n=@
+  ^-  tape
+  %-  prefix-hex
+  ?:  =(0 n)
+    "0"
+  %-  render-hex-bytes-minimal
+  (as-octs:mimes:html n)
```

Comment:
You really don't need this as octs. It seems fine to me to just call `(x-co:co 1)` here directly. That should also take care of the zero case.

Reply:
Thanks @Fang- as I said before I'm quite new to hoon and urbit, so I'd happily take a direct patch in hoon, just to be sure I understand.

Reply:
I think @Fang- is saying this: 

```suggestion
  ?:  =(0 n)
    "0"
  ((x-co:co 1) n)
```

and then you don't need the helper func below

Reply:
Done.